js/controllers/OfficersCtrl.js

```diff
@@ -26,6 +26,30 @@ angular.module('OfficersCtrl', []).controller('OfficersCtrl', ['$scope', '$sce',
             header: 'Current Officer Board',
             internal_title: 'Current Officer Board',
             body: 'Line Side: \n' +
+            '* Captain: Everest Orloff \n' +
+            '* First Lieutenant: Martin Smith \n' +
+            '* Second Lieutenant: Dianna Stuzhuk \n' +
+            '\nCivil Side:\n' +
+            '* President: Nathan Buckley\n' +
+            '* Vice President: Sandhya Vellayappan'
+        },
+        
+        {
+            header: 'Officer Board Summer-Fall 2020',

```

Comment:
Change to just Fall 2020, to better match the format of previous one semester boards. It's implied that the summer is included.